components/kyma-environment-broker/common/runtime/client.go

```diff
@@ -29,13 +27,12 @@ type client struct {
 
 // NewClient constructs and returns new Client for KEB /runtimes API
 // It takes the following arguments:
-//   - ctx  : context in which the http request will be executed
-//   - url  : base url of all KEB APIs, e.g. https://kyma-env-broker.kyma.local
-//   - auth : TokenSource object which provides the ID token for the HTTP request
-func NewClient(ctx context.Context, url string, auth oauth2.TokenSource) Client {
+//   - url        : base url of all KEB APIs, e.g. https://kyma-env-broker.kyma.local
+//   - httpClient : underlying HTTP client used for API call to KEB
+func NewClient(url string, httpClient *http.Client) Client {
```

Comment:
I see that implementation of the KEB client changed a bit, but you didn't introduce importing new version here:
https://github.com/kyma-project/control-plane/blob/294a1afd8f073957c80cc637681b406128ffbd38/tools/cli/pkg/command/taskrun.go#L207-L207

Are you planning to add it in separate PR after this one is merged?